Basic-Car-Maintenance/Assets.xcassets/basicGreen.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
+  "colors" : [
```

Comment:
for this color, you can change the appearance from "Any, Dark" to "none" which is fine cause it's a single color and the dark appearance doesn't change